app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -156,6 +199,95 @@ fun RunHikeBottomSheet(
+            ElevationGraph(
+                elevations = elevationData,
+                styleProperties =
+                    ElevationGraphStyleProperties(
+                        strokeColor = hikeColor, fillColor = hikeColor.copy(0.1f)),
+                modifier =
+                    Modifier.fillMaxWidth()
+                        .height(60.dp)
+                        .padding(4.dp)
+                        .testTag(TEST_TAG_ELEVATION_GRAPH))
+
+            // Progress details below the graph
+            Row(
+                modifier = Modifier.fillMaxWidth().padding(bottom = 20.dp),
+                horizontalArrangement = Arrangement.SpaceBetween) {
+                  Text(
+                      text = stringResource(R.string.run_hike_screen_zero_distance_progress_value),
+                      style = MaterialTheme.typography.bodyLarge,
+                      fontWeight = FontWeight.Bold,
+                      textAlign = TextAlign.Left,
+                      modifier = Modifier.padding(top = 8.dp),
+                  )
+                  Text(
+                      text = "23% complete",
+                      style = MaterialTheme.typography.bodyLarge,
+                      color = hikeColor,
+                      fontWeight = FontWeight.Bold,
+                      textAlign = TextAlign.Right,
+                      modifier = Modifier.padding(top = 8.dp).testTag(TEST_TAG_PROGRESS_TEXT),
+                  )
+                  Text(
+                      text =
+                          stringResource(R.string.run_hike_screen_distance_progress_value_format)
+                              .format(hikeRoute.totalDistance),
+                      style = MaterialTheme.typography.bodyLarge,
+                      fontWeight = FontWeight.Bold,
+                      textAlign = TextAlign.Right,
+                      modifier = Modifier.padding(top = 8.dp).testTag(TEST_TAG_TOTAL_DISTANCE_TEXT),
+                  )
+                }
+
+            val elevationGainString = hikeRoute.elevationGain.roundToInt().toString()
+            val hourString =
+                String.format(Locale.getDefault(), "%02d", (hikeRoute.estimatedTime / 60).toInt())
+            val minuteString =
+                String.format(
+                    Locale.getDefault(), "%02d", (hikeRoute.estimatedTime % 60).roundToInt())
+
+            DetailRow(
+                label = stringResource(R.string.run_hike_screen_label_current_elevation),
+                value = "50m")
+            DetailRow(
+                label = stringResource(R.string.run_hike_screen_label_elevation_gain),
+                value = "${elevationGainString}m")
+            DetailRow(
+                label = stringResource(R.string.run_hike_screen_label_estimated_time),
+                value =
+                    if (hikeRoute.estimatedTime / 60 < 1) "${minuteString}min"
+                    else "${hourString}h${minuteString}")
```

Comment:
Suggestion: move those to `strings.xml` as `"%dmin"` and `"%dh%d"`.

Reply:
done